plugins/woocommerce-admin/client/marketplace/components/discover/discover.tsx

```diff
@@ -5,11 +5,13 @@
 /**
  * Internal dependencies
  */
+import './discover.scss';
 
-export default function Discover() {
+export default function Discover(): JSX.Element {
 	return (
-		<div className="woocommerce-marketplace__content">
-			<h1>Discover</h1>
+		<div className="woocommerce-marketplace__discover">
+			<h1>Discover Our Favorites</h1>
+			<div className="woocommerce-marketplace__content__placeholder"></div>
```

Comment:
Agreed 100%. I've updated all styling to match this convention now. Side note, I've removed the placeholder since it's literally that - a placeholder and is actually no longer needed.